editor/store/selectors.js

```diff
@@ -1603,24 +1719,16 @@ export function getBlockListSettings( state, uid ) {
  *
  * @return {string[]|boolean} Blocks that can be nested inside the block with the specified uid, or true/false to enable/disable all types.
  */
-export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
-	if ( ! globallyEnabledBlockTypes ) {
-		return false;
-	}
-
-	const supportedNestedBlocks = get( getBlockListSettings( state, uid ), [ 'supportedBlocks' ] );
-	if ( supportedNestedBlocks === true || supportedNestedBlocks === undefined ) {
-		return globallyEnabledBlockTypes;
-	}
-
-	if ( ! supportedNestedBlocks ) {
-		return false;
-	}
+export function getSupportedBlocks( state, uid ) {
```

Comment:
`getSupportedBlocks` has to accept `globallyEnabledBlockTypes` to be backwards compatible. I removed it from the signature to avoid an unused variable linter warning. I've made this more explicit by just disabling the linter warning in d42964fdd 👍